src/components/AssetSearch/components/DefaultAssetList.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Asset } from '@shapeshiftoss/types'
+import { useMemo } from 'react'
+
+import { GroupedAssetList } from './GroupedAssetList/GroupedAssetList'
+
+export type DefaultAssetListProps = {
+  isPortfolioLoading: boolean
+  isPopularAssetIdsLoading: boolean
+  portfolioAssetsSortedByBalance: Asset[]
+  popularAssets: Asset[]
+  onClickItem: (asset: Asset) => void
+}
+
+export const DefaultAssetList = ({
+  isPortfolioLoading,
+  isPopularAssetIdsLoading,
```

Comment:
And that

Reply:
This flag comes from the hook `useGetPopularAssetsQuery` which we need to call in 1 place to prevent duplicate work for performance reasons. It made sense to make the portfolio assets follow the same pattern, though its not strictly necessary in the case of the portfolio loading flag

Reply:
`we need to call in 1 place to prevent duplicate work for performance reasons.`
If the rationale is a performance one then that's a non-issue - `useQuery` can be seen as a singleton of sorts...ish. As long a there is at least one listener, the query will be considered active and multiple listeners of the *same* query (which is the case here because of the wrapper) with the same options (which is also the case because of this wrapper hook) will leverage the query cache. 

Regardless, `useGetPopularAssetsQuery` wraps a query with a `staleTime` of `Infinity` so there wouldn't really be performance considerations here, as the query will be effectively considered stale - which leads me to another point, if the intent is to have this forever-cached, we should also `gcTime` it to `Infinity` to further ensure things don't get refetched (the query may be stale forever, but will be garbage -collected after the default 5mn, which means that regardless of it not being stale, it will still be refetched because the data is gone from the cache).

Reply:
@woodenfurniture q ^^

Reply:
Ah okay, it makes sense in that case. If you'd prefer we avoid drilling in loading flags and call the query lets do it :) Ideally we keep the actual data drilled through so the "quick access" assets can consume the same data. That is unless there is a clever way (and a reason) to abstract that also through use of a selector?

Reply:
As discussed, data is absolutely fine to keep as-is - ideally wouldn't have to and we could used a `reselect` selector with react-query's `select` property not to re-run computations more than we should, but happy to either keep it as-is, or keep it for a follow-up as this should be quite straightforward to implement